src/odemis/gui/cont/views.py

```diff
@@ -373,11 +376,19 @@ def _initialize_ovv_im(self, shape):
     def _on_merge_ratio_change(self, ratio):
         self.canvas.history_overlay.set_merge_ratio(ratio)
 
-    def on_stage_pos_change(self, p_pos):
+    def on_stage_pos_change(self, pos):
         """ Store the new position in the overview history when the stage moves,
         update the overview image """
 
-        # History
+        # Wait 0.5 seconds before adding the position to the history to get fewer
```

Comment:
Make this comment also more explicit. Something like: "If the stage hasn't moved within the next 0.5 s, we will considered it's stopped, and so will update the position. Without doing so, every stage position would be drawn, resulting in a very cluttered view." 

You might also want to explain why the `CallAfter()`, and why the `.Stop()`